recipes/eliot/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "eliot" %}
+{% set version = "1.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 973e5c9518b24a85d4ed2841882e32359ab5cdaf8389fc98dbd24866f052b84c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python > 3.4
```

Comment:
I am like 90% sure you need `python >3.4` here. I have gotten errors when the space is there for whatever reason.

Reply:
Thanks, fixed.